utils/symbols.stanza

```diff
@@ -63,76 +63,76 @@ public pcb-symbol altium-power-arrow-sym :
 
-  unit-line([[0.0, 0.0], [1.27, 0.0]])
-  unit-triangle([1.27, 0.0], [2.794, 0.0], 1.524)
-  unit-val([2.5, -0.5])
+  unit-line([[0.0, 0.0], [0.5, 0.0]])
+  unit-triangle([0.5, 0.0], [1.1, 0.0], 0.6)
+  unit-val([1.0, -0.2])
 
   preferred-orientation = PreferRotation([1])
 
 public pcb-symbol altium-power-circle-sym :
   name = "ALTIUM-POWER-CIRCLE"
   pin p[0] at unit-point(0.0, 0.0)
 
-  unit-circle([3.0 / 2.0 * 1.27, 0.0], 1.27 / 2.0)
-  unit-line([[0.0, 0.0], [1.27, 0.0]])
-  unit-val([2.5, -0.5])
+  unit-circle([3.0 / 2.0 * 0.5, 0.0], 0.5 / 2.0)
+  unit-line([[0.0, 0.0], [0.5, 0.0]])
+  unit-val([1.0, -0.2])
   
   preferred-orientation = PreferRotation([1])
 
 public pcb-symbol altium-power-bar-sym : 
   name = "ALTIUM-POWER-BAR"
   pin p[0] at unit-point(0.0, 0.0)
 
-  val ex = 3.0 / 2.0 * 1.27
+  val ex = 3.0 / 2.0 * 0.5
   unit-line([[0.0, 0.0], [ex, 0.0]])
-  unit-line([[ex, -1.27 / 2.0], [ex, 1.27 / 2.0]])
-  unit-val([0.5, 0.5])
+  unit-line([[ex, 0.5 / 2.0], [ex, 0.5 / 2.0]])
+  unit-val([0.2, 0.2])
 
   preferred-orientation = PreferRotation([1])
 
 public pcb-symbol altium-power-wave-sym :
   name = "ALTIUM-POWER-WAVE"
   pin p[0] at unit-point(0.0, 0.0)
 
-  val u = 0.5 * 1.27
+  val u = 0.5 / 2.0
   val ex = 4.0 * u
   unit-line([[0.0, 0.0], [ex, 0.0]])
   unit-line([[ex - u, u], [ex, 0.5 * u]])
   unit-line([[ex, 0.5 * u], [ex, -0.5 * u]])
   unit-line([[ex + u, (- u)], [ex, -0.5 * u]])
-  unit-val([1.5, -0.5])
+  unit-val([0.6, -0.2])
 
   preferred-orientation = PreferRotation([1])
 
 public pcb-symbol altium-power-gnd-power-sym :
   name = "ALTIUM-POWER-GND-POWER"
   pin p[0] at unit-point(0.0, 0.0)
 
-  val u = 1.27 / 2.0
+  val u = 0.5 / 2.0
   unit-line([[0.0, 0.0], [2.0 * u, 0.0]])
-  unit-line(0.1, [[2.00 * u, -1.40], [2.00 * u, 1.40]])
-  unit-line(0.1, [[2.67 * u, -1.08], [2.67 * u, 1.08]])
-  unit-line(0.1, [[3.33 * u, -0.77], [3.33 * u, 0.77]])
-  unit-line(0.1, [[4.00 * u, -0.46], [4.00 * u, 0.46]])
-  unit-val([2.5, -1.0])
+  unit-line(0.1, [[2.00 * u, -0.55], [2.00 * u, 0.55]])
+  unit-line(0.1, [[2.67 * u, -0.43], [2.67 * u, 0.43]])
+  unit-line(0.1, [[3.33 * u, -0.3], [3.33 * u, 0.3]])
+  unit-line(0.1, [[4.00 * u, -0.18], [4.00 * u, 0.18]])
+  unit-val([1.0, -0.4])
 
   preferred-orientation = PreferRotation([3])
 
 public pcb-symbol altium-power-gnd-signal-sym :
   name = "ALTIUM-POWER-GND-SIGNAL"
   pin p[0] at unit-point(0.0, 0.0)
 
-  unit-line([[0.0, 0.0], [1.27, 0.0]])
-  unit-triangle([1.27, 0.0], [1.5 * 2.794, 0.0], 1.5 * 1.524)
-  unit-val([2.5, -1.0])
+  unit-line([[0.0, 0.0], [0.2, 0.0]])
+  unit-triangle([0.2, 0.0], [1.5 * 1.1, 0.0], 1.5 * 0.6)
+  unit-val([0.4, -0.16])
 
   preferred-orientation = PreferRotation([3])
 
 public pcb-symbol altium-power-gnd-earth-sym :
   name = "ALTIUM-POWER-GND-EARTH"
   pin p[0] at unit-point(0.0, 0.0)
 
-  val u = 0.5 * 1.27
+  val u = 0.5 / 2.0
   val ex = 4.0 * u
   unit-line([[0.0, 0.0], [ex, 0.0]])
   unit-line([[ex, u], [ex, -1.0 * u]])
```

Comment:
Just below line 143 it divides to 0.2 in unit-val

Reply:
changed and used `altium-power-gnd-earth-sym` and see this. It is good.
![image](https://user-images.githubusercontent.com/107752351/233803762-452a65a4-56ce-480c-b3d6-5241d37131c8.png)